content/en/docs/spinnaker/using-dinghy.md

```diff
@@ -1097,3 +1253,221 @@ Delete all Webhook validations with the `--all` parameter.
+                }
+            ]
+        }
+    }
+}
+```
+
+If this configuration exists dinghy will send a notification to channel `slack-channel-good` and `slack-channel-both` if pipeline was rendered fine and a notifcation to `slack-channel-bad` and `slack-channel-both` if pipeline failed to be rendered additionaly to the one being send as default for all the notifications.
+
+## Repository Template processing
+
+Imagine you have a template and a couple of modules and dinghyfiles pointing at them, then you modify a module and this module is using Rawdata, at this moment the commited Rawdata is from the template repository, so there can be two possible scenarios:
+
+  - The Rawdata from the template repository is taken in order to render again all the dependent dinghyfiles. `repositoryRawdataProcessing = false`
+  - The Rawdata from the specific dinghyfile re-rendered is taken to render the dinghyfile. `repositoryRawdataProcessing = true`
+
+By default dinghy will use the Rawdata from the template repository, however you can enable the second behavior in which the Rawdata will be from the latest push done for that dinghyfile on the specific repository.
+
+**Operator**
+
+```yaml
+apiVersion: spinnaker.armory.io/v1alpha2
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  spinnakerConfig:
+    config:
+      armory:
+        dinghy:
+          repositoryRawdataProcessing: true
+          ... # Rest of config omitted for brevity
+```
+
+Then update the SpinnakerService with your updated manifest:
+
+```bash
+kubectl -n spinnaker apply -f spinnakerservice.yml
+```
+
+**Halyard**
+
+* **Enable**
+
+  ```bash
+  hal armory dinghy repository_only_rawdata_processing enable
+  ```
+
+* **Disable**
+
+  ```bash
+  hal armory dinghy repository_only_rawdata_processing disable
+  ```
+
+### Repository Template processing example
+
+Given the next scenario:
+
+`dinghyfile` in armory/my-repository
+``` json
+  {
+    "application": "processtemplate",
+    "pipelines": [
+      {
+        "application": "processtemplate",
+        "name": "my-pipeline-name",
+        "stages": [
+          {
+            "name": "{{ .RawData.repository.full_name }}",
+            "type": "wait",
+            "waitTime": 10
+          },
+          {{ module "stage.minimal.wait.module" }}
+        ]
+      }
+    ]
+}
+```
+
+`stage.minimal.wait.module` in armory/template-repo
+``` json
+{
+  "name": "{{ var "waitname" ?: "Wait module" }}",
+  "waitTime":  "{{ var "waitTime" ?: 10 }}",
+  "type": "wait"
+}
+```
+
+When you make a commit to change `stage.minimal.wait.module` in armory/template-repo all the dependend pipelines will be rendered again (in this case the dinghyfile from armory/my-repository). For that the string `{{ .RawData.repository.full_name }}` will be:
+
+- With `repositoryRawdataProcessing disabled` the result will be `armory/template-repo`
+- With `repositoryRawdataProcessing enabled` the result will be `armory/my-repository`
```

Comment:
is there a reason this is `repositoryRawdataProcessing disabled` /enabled instead of just using the actual config `repositoryRawdataProcessing=false` / true like we did above?

Reply:
Yes is the same configuration.